nzshm_common/grids/region_grid.py

```diff
@@ -41,5 +49,74 @@ def load(self):
         return self.grid.load()
 
 
-def load_grid(gri_name: str) -> Tuple[float, float]:
-    return RegionGrid[gri_name].load()
+def load_grid(grid_name: str) -> List[LatLon]:
+    """
+    Load values from a region grid as `LatLon` pairs.
+
+    Examples:
+        >>> from nzshm_common import grids
+        >>> grids.load_grid("NZ_0_1_NB_1_0")
+        [
+            LatLon(latitude=-46.1, longitude=166.4),
+            LatLon(latitude=-46.0, longitude=166.4),
+            LatLon(latitude=-45.9, longitude=166.4),
+            ...
+        ]
+    """
+    return RegionGrid[grid_name].load()
+
+
+def get_location_grid(location_grid_name: str, resolution: float = DEFAULT_RESOLUTION) -> Iterable[CodedLocation]:
```

Comment:
We'll hold off on merging this until we've had a chance to think about how to best support these use cases.

This will involve thinking about the resolutions of the RegionGrid enums, and more generally handling collections of locations.

Reply:
I will write up an issue for this to handle independently, so that this can be merged into pre-release.

Reply:
Written up as #44.